src/index.ts

```diff
@@ -28,9 +28,11 @@ const OptionDefaults: IOptions = {
 /**
  * A markdown-it plugin for implementing docutils style roles and directives.
  */
-export default function docutilsPlugin(md: MarkdownIt, options?: IOptions): void {
+export function docutilsPlugin(md: MarkdownIt, options?: IOptions): void {
```

Comment:
what was the reason fro this change?

Reply:
I was fighting with default exports in jest. This has no changes but allows you to use in jest (default export was undefined)

Reply:
ok cheers, perhaps add a comment above `export default`, that this is required by jest

Reply:
Left a note!